src/components/NavBar.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+interface NavProps {
+  pages: string[];
+}
+
+const Nav: React.FC <NavProps> = ({ pages }) => (
+  <nav>
+    <ul>
+      {pages.map(page => (
+        <li key={page}>
+          <Link to={`/${page}`}>{(page != "")?page:"home"} </Link>
```

Comment:
You can use NavLink instead of Link to make current link active (it adds 'active' style to the link)